system-tests/README.md

```diff
@@ -1,119 +1,230 @@
-## System tests
+# System tests
+
+The system tests copy a file from a provider to a consumer blob storage account.
+
+## Publish/Build Tasks
```

Comment:
THe MVD on `main` now also uses the IdentityHub, which should be included in the instructions as well.

Reply:
at this stage the docker-compose for the local deployment is relying on `nginx`, when the local deploy is updated, the doc should be updated to reflect  this